src/maven.ts

```diff
@@ -0,0 +1,16 @@
+import {addPath, debug, endGroup, info, startGroup} from '@actions/core';
+import {exec} from '@tradeshift/actions-exec';
+import {isAvailable, install} from './sonar-scanner';
+
+export async function run(version: string, args: string[]): Promise<void> {
+  if (!(await isAvailable(version))) {
+    await install(version);
+  }
```

Comment:
This installs the sonar-scanner cli I don't think you need that when running with maven?